docs/runbooks/dnsmasq.md

```diff
@@ -0,0 +1,10 @@
+# Vault
+
+## Overview
+dnsmasq is used as either a DHCP server or DHCP proxy server for PXE metal provisioning. Proxy mode can be opted-in through `make configure` or the `metal/roles/pxe_server/defaults/main.yml` file
```

Comment:
Nit: based on the [official website](https://thekelleys.org.uk/dnsmasq/doc.html), we can call it dnsmasq or Dnsmasq (at the start of the sentence)

```suggestion

Dnsmasq is used as either a DHCP server or DHCP proxy server for PXE metal provisioning. Proxy mode can be opted-in through `make configure` or the `metal/roles/pxe_server/defaults/main.yml` file.
```